test/e2e/e2e_test.go

```diff
@@ -57,6 +34,44 @@ func TestE2EFlow(t *testing.T) {
 		})
 	})
 
+	t.Run("verify MemberOperatorConfigs synced from ToolchainConfig to member clusters", func(t *testing.T) {
+		currentConfig := hostAwait.GetToolchainConfig()
+		expectedMemberConfiguration := currentConfig.Spec.Members.Default
+
+		t.Run("verify ToolchainConfig has synced status", func(t *testing.T) {
+			VerifyToolchainConfig(t, hostAwait, wait.UntilToolchainConfigHasSyncedStatus(ToolchainConfigSyncComplete()))
+		})
+		t.Run("verify MemberOperatorConfig was synced to member 1", func(t *testing.T) {
+			VerifyMemberOperatorConfig(t, hostAwait, memberAwait, wait.UntilMemberConfigMatches(expectedMemberConfiguration))
+		})
+		t.Run("verify MemberOperatorConfig was synced to member 2", func(t *testing.T) {
+			member2ExpectedConfig := testconfig.NewMemberOperatorConfig(testconfig.Webhook().Deploy(false))
+			VerifyMemberOperatorConfig(t, hostAwait, memberAwait2, wait.UntilMemberConfigMatches(member2ExpectedConfig.Spec))
+		})
+		t.Run("verify updated toolchainconfig is synced - go to unready", func(t *testing.T) {
+			// set the che required flag to true to force an error on the memberstatus (che is not installed in e2e test environments)
+			memberConfigurationWithCheRequired := testconfig.ModifyMemberOperatorConfig(memberAwait.GetMemberOperatorConfig(), testconfig.Che().Required(true))
+			hostAwait.UpdateToolchainConfig(testconfig.Members().Default(memberConfigurationWithCheRequired.Spec))
+
+			err := memberAwait.WaitForMemberStatus(
+				wait.UntilMemberStatusHasConditions(ToolchainStatusComponentsNotReady("[routes]")))
+			require.NoError(t, err, "failed while waiting for MemberStatus to contain error due to che being required")
+
+			_, err = hostAwait.WaitForToolchainStatus(
+				wait.UntilToolchainStatusHasConditions(ToolchainStatusComponentsNotReady("[members]"), ToolchainStatusUnreadyNotificationNotCreated()))
+			require.NoError(t, err, "failed while waiting for ToolchainStatus to contain error due to che being required")
+
```

Comment:
Just want to comment that in this test you don't verify only the logic of propagating/syncing the config between host&member in an isolated way. You also make some assumptions that the ToolchainStatus should fail as a result of:
* enabling che when Che is not installed (route is not available)
* propagating the memberstatus to toolchainstatus 

I know that it illustrates some unhappy real e2e scenario, but do we really need/want to do that in this particular case? wouldn't it be better/safer to just modify either `RefreshPeriod` or `HealthCheckPeriod`?

Reply:
Hmm, we could but how would we verify the change though?

Reply:
you update it in `ToolchainConfig` and wait until the `MemberOperatorConfig` is updated in the member. Or am I missing something?

Reply:
Yeah, that verifies that the MemberOperatorConfig is updated but I thought it would be good to also verify the new updated config takes effect, basically verifying the cache has been updated with the new config. Enabling Che when Che is not installed causes an effect we can easily verify. But if we think this is covered enough by unit tests or having it could cause problems then I'm okay to remove it.

Reply:
Btw, couldn't you just modify the `RefreshPeriod` of the `MemberStatus` for example to 1 second, and then check that the `LastUpdatedTime` is updated after one second?
It should make the test a bit faster and it wouldn't rely on some side effects like errors when Che is not properly configured.

I see that your e2e tests are green, so feel free to merge it and we can discuss/modify afterwards.